src/GoCSignature/GoCSignature.js

```diff
@@ -1,23 +1,25 @@
-import React from "react"
-import styled from "react-emotion"
-import PropTypes from "prop-types"
+import React from 'react'
+import styled from 'react-emotion'
+import PropTypes from 'prop-types'
 
-const FipFlag = styled("path")`
+const FipFlag = styled('path')`
   fill: ${props => props.flag};
 `
 
-const FipText = styled("path")`
+const FipText = styled('path')`
   fill: ${props => props.text};
-  transform: ${props => (props.transform ? props.transform : "translate(0 0)")};
+  transform: ${props => (props.transform ? props.transform : 'translate(0 0)')};
 `
 
 const Wordmark = ({
-  width = "10em",
-  flag = "#F00",
-  text = "#000",
-  lang = "en",
+  width = '10em',
+  flag = '#F00',
+  text = '#000',
+  lang = 'en',
 }) => (
   <svg
+    role="img"
+    aria-label="Government of Canada / Gouvernement du Canada"
```

Comment:
Is this going to be read out by a screen reader? I think it is, so we should probably save the user some time by only reading them the text for the current language. How about a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here:  `{lang === "en" ? "Government of Canada" : "Gouvernement du Canada"}`?
Also, is this something that is supposed to be descriptive ("the logo of the Government of Canada") or simply provide equivalent text?

Reply:
I can make that change. As for the text, I was taking it directly from the code I had observed on multiple GoC webpages. I can discuss with Paul and the team and get their opinion as well.